src/r2b2/minerva.py

```diff
@@ -0,0 +1,218 @@
+"""Minerva audit module."""
+import math
+from typing import List
+
+import click
+from scipy.stats import binom
+
+from r2b2.audit import Audit
+from r2b2.contest import Contest
+
+
+class Minerva(Audit):
+    """Minerva audit implementation.
+
+    A Minerva audit is a type of risk-limiting audit that accounts for round-by-round auditor
+    decisions. For a given sample size (in the context of a round schedule), the audit software
+    calculates a minimum number of votes for the reported winner that must be found in the sample
+    to stop the audit and confirm the reported outcome.
+
+    Attributes:
+        alpha (float): Risk limit. Alpha represents the chance that, given an incorrectly called
+        election, the audit will fail to force a full recount.
+        max_fraction_to_draw (float): The maximum number of ballots the auditors are willing to draw
+        as a fraction of the ballots in the contest.
+        rounds (List[int]): Cumulative round schedule.
+        min_winner_ballots (List[int]): Stopping sizes (or kmins) respective to the round schedule.
+        contest (Contest): Contest to be audited.
+    """
+
+    def __init__(self, alpha: float, max_fraction_to_draw: float, contest: Contest):
+        """Initialize a Minerva audit."""
+        super().__init__(alpha, 0.0, max_fraction_to_draw, True, contest)
+        self.min_sample_size = self.get_min_sample_size()
+        self.rounds = []
+        self.min_winner_ballots = []
```

Comment:
I believe these are initialized empty in the audit init on line 32. These won't cause problems, but is there a specific reason you included these in the Minerva init as well?

Reply:
No, there isn't a reason; these lines will be removed.